DSCResources/MSFT_xSQLServerScript/MSFT_xSQLServerScript.psm1

```diff
@@ -0,0 +1,173 @@
+﻿function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)]
+        [System.String]
+        $ServerInstance,
+
+        [parameter(Mandatory = $true)]
+        [System.String]
+        $SetFilePath,
+
+        [parameter(Mandatory = $true)]
+        [System.String]
+        $GetFilePath,
+
+        [parameter(Mandatory = $true)]
+        [System.String]
+        $TestFilePath,
+
+        [System.Management.Automation.PSCredential]
+        $Credential,
+
+        [System.String[]]
+        $Variable
+    )
+
+    Import-Module -Name SQLPS -WarningAction SilentlyContinue -ErrorAction Stop
```

Comment:
With the latest updates to SSMS the SQLPS module was renamed to SQLServer so baring this in mind should we not have a check in here for the change of module name for this along the lines of
```powershell
If (Get-Module SQLPS -ListAvailable) 
    {
        Write-Verbose "SQLPS Module Found"
        $module = Get-Module SQLPS -ListAvailable
    } 
    else 
    {
    Write-Verbose "SQLPS Module Not found - Using updated SQLServer module "
    $module = Get-Module SQLServer -ListAvailable
    }
    
Import-Module $module -WarningAction SilentlyContinue -ErrorAction Stop
```

Though this would need to be actioned in the same way across all of the resources where this is currently in place